api/types/swarm/common.go

```diff
@@ -17,7 +17,7 @@ type Meta struct {
 // Annotations represents how to describe an object.
 type Annotations struct {
 	Name   string            `json:",omitempty"`
-	Labels map[string]string `json:",omitempty"`
+	Labels map[string]string `json:"Labels"`
```

Comment:
I don't think the `omitempty` here is related to #24631. I don't think there's any harm in continuing to omit the `Labels` field when it's empty.